請問下列程式執行後的結果為何？為什麼？

```javascript
setTimeout(() => console.log("Timeout"), 0);

Promise.resolve().then(() => console.log("in promise"));

console.log("outside");
```
